src/Nowruz/modules/layout/components/headerNavBar/index.tsx

```diff
@@ -42,6 +41,8 @@ const HeaderNavBar: React.FC<HeaderNavBarProps> = ({ setOpen, logout }) => {
     setSearchTerm,
   } = useHeaderNavBar();
 
+  const searchPlaceholder = window.location.pathname === '/jobs' ? 'Search by title, skill or organization' : 'Search';
```

Comment:
this will work, but can you use useLocation() hook from react router dom? Thanks